librz/include/rz_bin.h

```diff
@@ -744,6 +744,20 @@ struct rz_bin_reloc_storage_t {
 	size_t target_relocs_count;
 }; // RzBinRelocStorage
 
+typedef struct rz_bin_reloc_formular_symbols_t {
```

Comment:
I don't think it should be in the `rz_bin.h`. Could you please move it back to the `bin_elf.inc`? It's ELF-specific, not needed for anything else.